rke/k8s_rke_system_images.go

```diff
@@ -4364,6 +4364,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.6"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.13"),
 		},
+		// Enabled out of band post v2.5.6
+		// Reminder: This template contains ACI images which aren't in templates for v2.4.x
+		"v1.20.2-rancher2-1": {
+			Etcd:                      m("rancher/coreos-etcd:v3.4.14-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.20.2-rancher2"),
+			Alpine:                    m("rancher/rke-tools:v0.1.70"),
```

Comment:
v0.1.71 now to make 20h2 work

Reply:
@luthermonson thanks, will get it updated